r2-core/src/main/java/com/linkedin/r2/transport/http/client/AsyncPoolImpl.java

```diff
@@ -209,11 +231,13 @@ public AsyncPoolImpl(String name,
     _lifecycle = lifecycle;
     _maxSize = maxSize;
     _idleTimeout = idleTimeout;
+    _waitTimeout = waiterTimeout;
```

Comment:
Please name the parameter and class variable consistently. _waitTimeout vs. waiterTimeout.